lib/Settings/Admin.php

```diff
@@ -49,14 +56,17 @@ public function getForm(): TemplateResponse {
 			$id = (int)$oauthClientInternalId;
 			$clientInfo = $this->oauthService->getClientInfo($id);
 		}
-
+		$groupFolderStatusInformation = $this->openProjectAPIService->getGroupFolderSetupInformation();
 		$adminConfig = [
 			'openproject_client_id' => $clientID,
 			'openproject_client_secret' => $clientSecret,
 			'openproject_instance_url' => $oauthUrl,
 			'nc_oauth_client' => $clientInfo,
 			'default_enable_navigation' => $this->config->getAppValue(Application::APP_ID, 'default_enable_navigation', '0') === '1',
 			'default_enable_unified_search' => $this->config->getAppValue(Application::APP_ID, 'default_enable_unified_search', '0') === '1',
+			'app_password_set' => $this->openProjectAPIService->hasAppPassword(),
+			'group_folder_status' => $groupFolderStatusInformation,
```

Comment:
Will it make sense here too?
```suggestion
			'project_folder_status' => $groupFolderStatusInformation,
```